server/routes/index.ts

```diff
@@ -32,7 +32,7 @@ function createRouter(repositories: Repositories) {
     });
 
     router.get(nav.home.href, (req, res) => {
-        res.render('profile', { ...nav.locals(req), user: req.user });
+        res.render('profile', { ...nav.locals(req), user: req.params.userId });
```

Comment:
`nav.locals` already sets a `user` local, which contains the currently logged in user if any. So that local should probably be renamed to `currentUser`, if you want to use `user` for this route. Or some other way to resolve the conflict